src/cachingreader.cpp

```diff
@@ -162,66 +159,90 @@ void CachingReader::freeAllChunks() {
     m_allocatedChunks.clear();
     m_mruChunk = NULL;
 
-    QSet<Chunk*> reserved = QSet<Chunk*>::fromList(m_chunksBeingRead.values());
+    for (QVector<Chunk*>::const_iterator it = m_chunks.constBegin();
+         it != m_chunks.constEnd(); ++it) {
```

Comment:
This looks bizarre to me:

``` 
if (asdf && qwer && 
        zxcv) {
}
```
I accept both styles even though this is not valid Google style because you like this one and I don't care too much about style these days unless it's egregious.